pkgs/misc/vim-plugins/default.nix

```diff
@@ -1565,4 +1565,36 @@ rec {
 
   };
 
+  vim-github-colorscheme = buildVimPluginFrom2Nix { # created by nix#NixDerivation
+    name = "vim-colorscheme-2015-03-09";
```

Comment:
That came out of the generator like that too. The best is probably to drop the package altogether, the github colors are included in the vim-colorschemes package anyway.